docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in the system. Since this is a BETA version we only
+support tags for policies between meshObjects.
+
+## Create Policies
+
+Policy creation is only possible for a user who has a partner admin role right. There is a 'Create Policy' button top
+right on the policies screen. After clicking on it, there is a redirection to the screen of Create Policy where all
+details need to be filled up. To create the policy, you need to add the policy name in the form. Then you need to add
+left and right meshObjects from the dropdown menu. For ease of use, we already restricted the object combinations which
+do not make sense. You can choose only possible object combinations. Once you select both objects, the section for
+adding rules will appear where you can select the tag of selected meshObject.
+
+### Working explanation
+
+You can choose a possible tag for your organization for a particular meshObject. E.g. if you choose meshProject and
+meshLandingZone, and rule applied on environment tag with operator 'in' (which is the only operator we support now).
+Operator 'in' works like an intersection between the objects. Then you can assign a landing zone with prod environment
+to a project which has prod environment tag. If a project has three tags - dev, test, qa then that landing zone cannot
+be assignable. There has to be at least one rule assigned for policy.
+
+## Edit Policies
+
+In the policy item list, there is an edit policy button in the 'Actions' column which redirects to the Edit Policy
+screen where one can update the policy.
+
+## Delete policies
+
+In the policy item list, there is a delete policy button in the 'Actions' column. On clicking, it waits for 5 seconds
```

Comment:
```suggestion
In the policy item list, there is a delete policy button in the 'Actions' column. On clicking, it waits for 5 seconds
```
What is it?

Reply:
It explains the delete policy process. Action around delete policy button.